### DISCUSSION TITLE
django defult value not  set<title>

### DISCUSSION THREAD

**Discussion by yashzadafiya5** [community/community#39500] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

how i solve please suggest django in create data base in some bug defult value is not set only seen None

**Comment by jfullstackdev** [comment]
maybe this can help 

https://medium.com/botify-labs/django-and-its-default-values-c21a13cff9f

**Comment by avinashkr-ai** [comment]
`username = models.CharField(max_length=100, default=fun)`

`def fun():
    return 'yashzadafiya5'
`
